client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/toggle-activate-monitoring/index.tsx

```diff
@@ -153,19 +162,47 @@ export default function ToggleActivateMonitoring( {
 		);
 	}
 
+	const displayablePopover = shouldDisplayUpgradePopover ? (
+		<UpgradePopover
+			context={ statusContentRef.current }
+			isVisible={ showTooltip }
+			position="bottom left"
+			onClose={ handleHideTooltip }
+		/>
+	) : (
+		<>
+			{ tooltip && (
+				<Tooltip
+					id={ tooltipId }
+					context={ statusContentRef.current }
+					isVisible={ showTooltip }
+					position="bottom"
+					className="sites-overview__tooltip"
+				>
+					{ tooltip }
+				</Tooltip>
+			) }
+		</>
```

Comment:
[Non-blocking] The usage of Fragment here (i.e., `<>`) isn't strictly necessary:

```suggestion
		{ tooltip && (
			<Tooltip
				id={ tooltipId }
				context={ statusContentRef.current }
				isVisible={ showTooltip }
				position="bottom"
				className="sites-overview__tooltip"
			>
				{ tooltip }
			</Tooltip>
		) }
```